### DISCUSSION TITLE
Units for pathway expression

### DISCUSSION THREAD

**Discussion by Igor0411** [jackbibby1/SCPA#14] [Q&A] [answered]
Hello Jack! Thank you very much for such a great tool but one moment for me is unclear. 
I got through the papers you mentioned on the main SCPA page but due to lack of knowledge in math I wasn't able to understand in what units pathways expression measures and further compares (I mean, we've got FC somehow). 
P.S. Sorry for such a silly question.

**Comment by jackbibby1** [comment, marked as answer]
Hey.

No worries at all. In the SCPA output, the Qval that we give is a transformed adjusted pval, and is the metric that we recommend using. You can consider that the higher the Qval, the more differentially regulated the pathway is between conditions. If you're doing a two-sample comparison, we also give a fold change value that's calculated from average pathway expression in population1 - population2 (it's a running sum of mean changes in gene expression for that pathway), so a negative value means the pathway will be higher in population2. Hope that's what you were looking for?

I'll also add some more detailed explanation into the help documentation -- `?compare_pathways()` -- to give a bit more information on how this is calculated. 

Jack

**Reply by jackbibby1** [reply]
OK -- I've added some detail about the FC calculation if you run `?compare_pathways()` (after updating the package), and I've also added some more explanation on the quick start page of the SCPA site here: https://jackbibby1.github.io/SCPA/articles/quick_start.html.

**Reply by Igor0411** [reply]
Thank You very much! That was exactly what I was looking for))
And also thank you so much for such a quick response!

**Comment by SuduSharma** [comment]
Hi Jack,
Thank you very much for making SCPA—many thanks for updating the seurat_extract function to include seuratv5. I followed your vignette for a system-level comparison between the control and diseased state. Is it reasonable to expect the q values from the saved scpa_out csv  to be represented in the heatmap? I do not see a match in the heatmap.

**Reply by jackbibby1** [reply]
Hey,

Are you meaning [this tutorial](https://jackbibby1.github.io/SCPA/articles/disease_comparison.html)? If you're following the same code in the tutorial then there should be the same results, hopefully. Did you just follow all the code on the page? Can you show me the results you got?

Thanks
Jack

**Reply by jackbibby1** [reply]
FYI -- I just re-ran the analysis using the code on the SCPA tutorial page and I get the same result. You may get slightly different results in the row/column clustering because ComplexHeatmap uses kmeans for this, but the actual qvals should be very similar

**Comment by SuduSharma** [comment]
Hi  Jack
Yes, I was using the tutorial with my data. You were right about the
complex heatmap doing something. My column clustering used to get messed up
(column names would show a cell cluster with meager differential pathway
enrichment as the best hit. It might just be me. However, when I took out
most custom parameters from Heatmap, it just ordered perfectly.
Thank you again, and sorry for the delay in answering.
Sudu
